### PR TITLE
Move sanity checks to separate crowbar lib

### PR DESCRIPTION
**Pull request by rsalevsky**
Depends on https://github.com/crowbar/crowbar-core/pull/61

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM

**Comment by vuntz** [comment]
FWIW, I don't think this can be merged until the install process is: "open your browser and click the next buttons".

**Comment by rsalevsky** [comment]
Depends on: https://github.com/crowbar/crowbar-core/pull/61

**Comment by MaximilianMeister** [comment]
assuming it works :+1:  :)

**Comment by vuntz** [comment]
> FWIW, I don't think this can be merged until the install process is: "open your browser and click the next buttons".

Would it be possible to add a rake task to call the checks? And then, while we still depend on install-suse-cloud, we would simply call the rake task here.

**Comment by vuntz** [comment]
s/stat/start/ in commit message

File: scripts/install-chef-suse.sh [hunk 1]
```diff
@@ -304,31 +304,6 @@ case "$rootpw" in
...
-if [ -z "$FQDN" -o -z "$DOMAIN" ]; then
-    die "Unable to detect fully-qualified hostname. Aborting."
-fi
-
-if ! resolved=$(getent ahosts $FQDN 2>/dev/null); then
-    die "Unable to resolve hostname $FQDN via host(1). Please check your configuration of DNS, hostname, and /etc/hosts. Aborting."
-fi
-
-IPv4_addr=$( echo "$resolved" | awk '{ if ($1 !~ /:/) { print $1; exit } }' )
-IPv6_addr=$( echo "$resolved" | awk '{ if ($1  ~ /:/) { print $1; exit } }' )
```
Comment (vuntz) [inline scripts/install-chef-suse.sh]: -1: all the variables that were defined are actually used later in the script, so we need to keep them defined...

Comment (rsalevsky) [inline scripts/install-chef-suse.sh]: Fixed, the IPv6 is not used.

**Comment by jdsn** [comment]
:+1:

**Pull request merged**